adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1780,6 +1792,65 @@ private void checkInternetPermission() {
     }
 
     /**
+     * To verify if App gives permissions to AccountManager to use broker.
+     *
+     * @throws UsageAuthenticationException
+     * @return boolean If the GET_ACCOUNTS, MANAGE_ACCOUNTS, USE_CREDENTIALS permissions are granted in the Manifest.xml
+     */
+    private boolean verifyManifestPermissions() throws UsageAuthenticationException {
+        if(Build.VERSION.SDK_INT <= 22) {
```

Comment:
on API 23 and above, if user don't request GET_ACCOUNTS permission, then are we allowing it?